app/views/social_previews/articles/shecoded.html.erb

```diff
@@ -1,47 +1,103 @@
 <style>
   body {
-    <% if @article.cached_tag_list.include?("theycoded") %>
-      background-color: #cabde6;
-    <% else %>
-      background: url(<%= asset_path "SheCodedBackground.png" %>);
-      background-color: #cabde6;
-      background-repeat: repeat;
-    <% end %>
+    margin: 0;
+    padding: 0;
+    font-family: "Roboto", "Helvetica Neue", Helvetica, Arial, "Lucida Grande", sans-serif;
+    background: white;
   }
   .preview-div-wrapper {
-    background: white;
-    width: 80vw;
-    height: 35vw;
-    margin: 2.5vw auto 2.5vw;
-    padding: 4vw;
-  }
-  .avatar {
-    margin-left: auto;
-    margin-right: auto;
-    margin-bottom: 2.5vw;
-    width: 12vw;
-  }
-  img {
-    border-radius: 12vw;
-    border: .75vw solid #70589b;
-    height: 12vw;
-    width: 12vw;
-  }
-  .title {
-    font-size: <%= 7 - @article.title.size / 40 %>vw;
-    margin: auto;
-    width: 80%;
-    color: #70589b;
-    text-align: center;
-    font-family: "HelveticaNeue-CondensedBold", "HelveticaNeueBoldCondensed", "HelveticaNeue-Bold-Condensed", "Helvetica Neue Bold Condensed", "HelveticaNeueBold", "HelveticaNeue-Bold", "Helvetica Neue Bold", "HelveticaNeue", "Helvetica Neue", 'TeXGyreHerosCnBold', "Helvetica", "Tahoma", "Geneva", "Arial Narrow", "Arial", "Roboto Condensed", sans-serif;
+    overflow: hidden;
+    position: relative;
+  }
+  .preview-div {
+    border-radius: 25px 25px 0 0;
+    background: url(/assets/shecoded-background.jpg);
+    background-position: 50% 50%;
+    background-size: 100%;
+    overflow: hidden;
+    width: 94vw;
+    margin: 2.25vw auto 5vw;
+  }
+  .preview-article {
+    margin: 25px;
+    border-radius: 20px 20px 0 0;
+    background: #FFFFFF;
+    box-shadow: 5px 5px 0px #8220B0;
+    border: 1px solid #8220B0;
+    position: relative;
+    min-height: 200px;
+  }
+  .tag-header {
+    color: #8220B0;
+    margin: 20px;
+    font-size: 15px;
+    line-height: 18px;
+  }
+  .title-area h1 {
+    font-style: normal;
+    font-weight: 500;
+    font-size: 36px;
+    line-height: 42px;
+    display: flex;
+    align-items: center;
+    letter-spacing: -0.02em;
+    margin: 0 20px;
+    color: #8220B0;
+    padding-bottom: 30px;
+  }
+  .preview-user {
+    color: #8220B0;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 15px;
+    line-height: 18px;
+    display: flex;
+    align-items: center;
+  }
+  .preview-user-info {
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+  .preview-user img {
+    border: 1px solid #8220B0;
+    box-sizing: border-box;
+    border-radius: 4.5vw;
+    margin: 20px;
```

Comment:
For this image, we don't have a full `20px` margin wrapping around because the right edge is closer to the name. I think that we can write:

`margin: 20px 8px 20px 20px;`